JobManagerExample/JobManagerExample/JobManagerExampleView.swift

```diff
@@ -0,0 +1,322 @@
+                isAddingOfflineMapJob = true
+                Task {
+                    do {
+                        jobManager.jobs.append(
+                            try await makeNapervilleOfflineMapJob()
+                        )
+                    } catch {
+                        Logger.jobManagerExample.error("Error creating offline map job: \(error, privacy: .public)")
+                    }
+                    isAddingOfflineMapJob = false
+                }
+            } label: {
+                Label("Take Map Offline", systemImage: "plus")
+            }
+            .disabled(isAddingOfflineMapJob)
+            
+            Divider()
+            
+            Button {
+                jobManager.jobs.removeAll()
+            } label: {
+                Label("Remove All Jobs", systemImage: "trash")
+            }
+            
+            Button {
+                jobManager.jobs.removeAll {
+                    $0.status == .failed || $0.status == .succeeded
+                }
+            } label: {
+                Label("Remove All Completed Jobs", systemImage: "trash")
+            }
+        } label: {
+            Text("Jobs")
+        }
+    }
+}
+
+/// A view that displays data for a job.
+private struct JobView: View {
+    /// The job that this view shows data for.
+    var job: Job
+    /// The job's error in case of failure.
+    @State var error: Error?
+    /// The job's satus.
+    @State var status: Job.Status
+    /// The latest job message.
+    @State var message: JobMessage?
+    
+    /// Initializer that takes a job for which to show the data for.
+    init(job: Job) {
+        self.job = job
+        status = job.status
+    }
+    
+    /// A Boolean value indicating if the progress is showing.
+    var isShowingProgress: Bool {
+        job.status == .started || job.status == .paused
+    }
+    
+    /// A Boolean value indicating if the cancel button is showing.
+    var isShowingCancel: Bool {
+        job.status == .started || job.status == .paused
+    }
+    
+    /// A string for the job's type.
+    var jobType: String {
+        "\(type(of: job))"
+    }
+    
+    var body: some View {
+        VStack (alignment: .leading, spacing: 5) {
+            Text(jobType)
+            Text(status.displayText)
+                .font(.footnote)
+            if let error = error {
+                Text(error.localizedDescription)
+                    .font(.footnote)
+                    .foregroundColor(.secondary)
+                    .lineLimit(1)
+            } else {
+                if let message = message {
+                    Text(message.text)
+                        .font(.footnote)
+                        .foregroundColor(.secondary)
+                        .lineLimit(1)
+                }
+                if isShowingProgress {
+                    ProgressView(job.progress)
+                        .progressViewStyle(.linear)
+                        .padding(.vertical)
```

Comment:
<img width="352" alt="Screenshot 2023-08-31 at 10 28 01" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/a72f0314-8b6b-469f-ba81-4f60eee00bcc">

There seems to be an item count (the "0 of 1") thats generating as apart of the progress view. It seems like it doesn't need to be there but I haven't found a way to get rid of it.

Reply:
I don't think there is a way unless we wrap the NSProgress and expose it out again.